back/app/controllers/web_api/v1/files_controller.rb

```diff
@@ -84,6 +84,14 @@ def create
     end
   end
 
+  def update
+    if @file.update(file_params)
+      render json: WebApi::V1::FileSerializer.new(@file, params: jsonapi_serializer_params).serializable_hash, status: :ok
+    else
+      render json: { errors: transform_carrierwave_error_details(@file.errors, :file) }, status: :unprocessable_entity
+    end
+  end
```

Comment:
Is this only here so we can update the ordering? Wonder if it's worth separating the `file_params` method into `create_file_params` and `update_file_params` to only allow ordering for update?